### DISCUSSION TITLE
What about search message?

### DISCUSSION THREAD

**Discussion by itaishalom** [flyerhq/flutter_chat_ui#147] [Ideas & Requests]
Is it an option?

**Comment by demchenkoalex** [comment]
Yes, totally. Since you provide all the data, you can make a search mechanism yourself (e.g. add a search widget on top of the chat one, or in the navigation bar) and search for the message. I guess we only miss some method to scroll to the index of that message in the chat.

**Reply by itaishalom** [reply]
Maybe I'll implement it on the library and PR you. Please PM if you plan to do it yourselfs
